public/stylesheets/site/2.0/21-userstuff.css

```diff
@@ -1,10 +1,11 @@
 /* ==USERSTUFF: displaying content inputted by a user into a textarea.*/
 
 .userstuff {
-    word-wrap: break-word;
+  word-wrap: break-word;
 }
 
-.userstuff p, .userstuff details {
+.userstuff p,
```

Comment:
Could you keep the selectors on one line for consistency with our other stylesheets, please?

Reply:
Sorry for being unclear -- I meant all the selectors throughout the file, not just these few lines.

Reply:
You were not at all unclear! I should just stop looking at code after 10PM. Should be fixed.